DevelopGitHub/GitHubExtension/GitHubExtension.Templates/ExtensionMethods/HttpResponseMessageExtension.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using GitHubExtension.Templates.CommunicationModels;
+using GitHubExtension.Templates.Exceptions;
+using Newtonsoft.Json;
+
+namespace GitHubExtension.Templates.ExtensionMethods
+{
+    public static class HttpResponseMessageExtension
+    {
+        public static async Task<string> GetTemplatesContent(this HttpResponseMessage message)
```

Comment:
Please don't mixed tech complexity related to await message.Content.ReadAsStringAsync() with getting GitHubTemplatesModel.